.github/workflows/pull_request.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: c-hive/gha-yarn-cache@v1
       - uses: actions/setup-node@v1
         with:
-          node-version: 12.x
+          node-version: 14.x
```

Comment:
I think we should bump to at least 16.x?

Reply:
See next comment